forem/forem
# Redesign sign-up and login pages

## What type of PR is this? (check all applicable)


## Description


Redesign sign-up and login pages and mainly include ToC, PP and CoC links.

## Related Tickets & Documents



- Related Issue #19988

## QA Instructions, Screenshots, Recordings

**How to test?**
Simply visit, `/enter` and `/enter?state=new-user` pages. Please check all internal links too.

### Output
<img width="432" alt="Screenshot 2023-09-08 at 2 37 45 PM" src="https://github.com/forem/forem/assets/9396084/a726803f-f9d6-4063-8087-a5bff8404090">
<img width="433" alt="Screenshot 2023-09-08 at 2 37 56 PM" src="https://github.com/forem/forem/assets/9396084/c303373d-5324-4dd1-98dd-e5727032ef2d">
<img width="431" alt="Screenshot 2023-09-08 at 2 38 12 PM" src="https://github.com/forem/forem/assets/9396084/d5a1e502-2669-479f-a64f-f66e8cd95430">
<img width="1512" alt="Screenshot 2023-09-08 at 2 38 27 PM" src="https://github.com/forem/forem/assets/9396084/19c1240d-469d-48bb-ab63-07edf39cdb2a">
<img width="1512" alt="Screenshot 2023-09-08 at 2 38 39 PM" src="https://github.com/forem/forem/assets/9396084/f067acfb-076a-4c9a-b68d-88b364a6c76c">
<img width="1512" alt="Screenshot 2023-09-08 at 2 38 51 PM" src="https://github.com/forem/forem/assets/9396084/d66549e7-8823-4c68-9ff4-a36b2d835daa">


<img width="444" alt="Screenshot 2023-09-08 at 2 47 20 PM" src="https://github.com/forem/forem/assets/9396084/5c7eb9c6-bc76-40cf-830c-285e3c83630a">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Added/updated tests?
